website/docs/docs/collaborate/explore-projects.md

```diff
@@ -147,7 +148,15 @@ From the sidebar, you can browse your project's resources, its file tree, and th
 - **File Tree** tab &mdash; All resources in the project organized by the file in which they are defined. This mirrors the file tree in your dbt project repository.
 - **Database** tab &mdash; All resources in the project organized by the database and schema in which they are built. This mirrors your data platform's structure that represents the [applied state](/docs/dbt-cloud-apis/project-state) of your project. 
 
-<Lightbox src="/img/docs/collaborate/dbt-explorer/example-search-sidebar.png" title="Example of tabs in sidebar" />
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-tabs-sidebar.png" title="Example of tabs in sidebar" />
+
+## Open in IDE
+
+If you have been assigned a [developer license](/docs/cloud/manage-access/about-user-access#license-based-access-control), you can open the resource in the [IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) directly from Explorer. The IDE opens, for example, the corresponding SQL and YAML files of a model. This feature is available from the [full lineage graph](#example-of-full-lineage-graph) and the [resource's details view](#example-of-model-details). 
```

Comment:
```suggestion
If you have been assigned a [developer license](/docs/cloud/manage-access/about-user-access#license-based-access-control), you can open the resource in the [IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) directly from Explorer. The IDE opens the all corresponding files for the model. This will include the model's SQL or Python definition, as well as any YML file that includes an entry for that model. This feature is available from the [full lineage graph](#example-of-full-lineage-graph) and the [resource's details view](#example-of-model-details). 
```